lib/collaborators.js

```diff
@@ -6,7 +6,8 @@ const TSC_TITLE = '### TSC (Technical Steering Committee)';
 const TSCE_TITLE = '### TSC Emeriti';
 const CL_TITLE = '### Collaborators';
 const CLE_TITLE = '### Collaborator Emeriti';
-const CONTACT_RE = /\* +\[(.+?)\]\(.+?\) +-\s\*\*(.+?)\*\* +&lt;(.+?)&gt;/mg;
+const CONTACT_RE =
+  /\* +\[(.+?)\]\(.+?\) +-\s\*\*(.+?)\*\* +(?:&lt;|<)(.+?)(?:&gt;|>)/mg;
```

Comment:
don't we need to include the backslash in the regexp?

Reply:
I mean, we clearly don't, because the tests pass, but I don't understand how it works ^^

Reply:
I'm not sure I understand which backslash you are talking about?

Reply:
The one you had to add in the readme to escape `<`

Reply:
It doesn't work. I thought I ran it against the main branch, but I guess I didn't.

Reply:
Nice catch, @targos. Fixed.